package-res/reportviewer/paginationTemplate.html

```diff
@@ -238,6 +238,10 @@
     );
   }
 
+  function numberWithCommas(x) {
+    return x.toString().replace(/\B(?=(\d{3})+(?!\d))/g, ",");
+  }
```

Comment:
If you want to format a number with thousands indicators, I would suggest you use locale aware code for that. 

For IE11 and similar modern browsers you can use "Number.prototype.toLocaleString()" (use the "useGroupingSeparator flag). If that method is not available (old browser etc.) you can fall back to your RegExp solution. But get it localized by default. 

Otherwise you will be surprised that not everyone uses comma for this purpose. Not everyone uses the US style system. (Given Russia and Belarus use comma as decimal separator lets not make everything conform to the English :) )